### ISSUE TITLE
Can you share training commands for Alexnet?

### ISSUE THREAD

**Issue by YouhuiBai** [dmlc/gluon-cv#1221]
I can't find training command and log for Alexnet in gluon Model Zoo, can someone share the training command or hyper-parameters to gain the accuracy about 54.92 as presented [here](https://gluon-cv.mxnet.io/model_zoo/classification.html#others), thanks a lot!

**Comment by hetong007** [comment]
One should be able to obtain the AlexNet model with commands and scripts from here: https://github.com/apache/incubator-mxnet/tree/master/example/image-classification

**Comment by YouhuiBai** [comment]
@hetong007 Thanks, but when I trained the AlexNet model using train_imagenet.py in this directory, the accuracy exceeds 90% when training over 90 epochs, maybe some hyper-parameters are not set correctly. The following is my command for training AlexNet, the default optimizer is momentum (0.9) SGD, learning rate scheduler is step and weight initializer is mxnet.init.Normal(). Are there any suggestions?
```txt
python train_imagenet.py --network alexnet --gpus 0 --lr 0.02 --lr-factor 0.1 --wd 0.0005 --batch-size 512 --data-train /data/trainData/traindata.rec --data-train-idx /data/trainData/traindata.idx --num-examples 1281167
```

**Comment by hetong007** [comment]
1. Are you setting the parameters according to the original paper?
2. What is the validation accuracy?

**Comment by YouhuiBai** [comment]
@hetong007 I set the hyper-parameters the same as Caffe except the weight initializer, the initializers are different among layers in Caffe, but Mxnet only takes Normal to initialize all parameters. The same hyper-parameters set in Pytorch will bring a correct accuracy about 56%, I guess that there are some implicit hyper-parameters in Mxnet. I don't open the accuracy validation before, but the opened one is in progress.

**Comment by hetong007** [comment]
You can initialize each layer in model definition: https://github.com/dmlc/gluon-cv/blob/master/gluoncv/model_zoo/simple_pose/simple_pose_resnet.py#L63

**Comment by YouhuiBai** [comment]
@hetong007 Thank you very much, I am trying...
